homeassistant/components/spotify/media_player.py

```diff
@@ -70,19 +71,29 @@
 
 BROWSE_LIMIT = 48
 
+MEDIA_TYPE_SHOW = "show"
```

Comment:
If `show` is a podcast shouldn't we use `podcast` instead to be consistent and just map show internally?

Reply:
For newly proposed field `media_type` I agree, it should be podcast, but `media_content_type` is just for internal use of the integration.